measurements/cli/__init__.py

```diff
@@ -26,6 +26,12 @@ def shell():
 
         code.interact(banner='', local=current_app.make_shell_context())
 
+@cli.command()
+@with_appcontext
+def create_tables():
+    from measurements.database import Base
+    from measurements import models
```

Comment:
Is `models` here imported for some side effect?

Reply:
Code moved nearly verbatim from `database.py`

Reply:
Cool: the `@cli.command` is such that `create-tables` is a command.

Reply:
yeah the goal of importing models in here is so that the classes are created (which have sqlalchemy metaclassing magic to register something inside of the base class) so that we can then run `create_all`